python/opae.admin/opae/admin/tools/pci_device.py

```diff
@@ -0,0 +1,106 @@
+#! /usr/bin/env python
+# Copyright(c) 2020, Intel Corporation
```

Comment:
2020-2021 ?

Reply:
2020-2021 or 2021?

Reply:
I think either 2020-2021 or 2021 is fine because it should be related to when the code was "published"